stake-pool/cli/src/main.rs

```diff
@@ -863,145 +867,148 @@ fn command_list(config: &Config, stake_pool_address: &Pubkey) -> CommandResult {
     let sol_withdraw_authority = stake_pool
         .sol_withdraw_authority
         .map_or("None".into(), |pubkey| pubkey.to_string());
+    if config.output_format == OutputFormat::Display || config.output_format == OutputFormat::DisplayVerbose {
+        if config.verbose {
+            println!("Stake Pool Info");
+            println!("===============");
+            println!("Stake Pool: {}", stake_pool_address);
+            println!("Validator List: {}", stake_pool.validator_list);
+            println!("Manager: {}", stake_pool.manager);
+            println!("Staker: {}", stake_pool.staker);
+            println!("Depositor: {}", stake_pool.stake_deposit_authority);
+            println!("SOL Deposit Authority: {}", sol_deposit_authority);
+            println!("SOL Withdraw Authority: {}", sol_withdraw_authority);
+            println!("Withdraw Authority: {}", pool_withdraw_authority);
+            println!("Pool Token Mint: {}", stake_pool.pool_mint);
+            println!("Fee Account: {}", stake_pool.manager_fee_account);
+        } else {
+            println!("Stake Pool: {}", stake_pool_address);
+            println!("Validator List: {}", stake_pool.validator_list);
+            println!("Pool Token Mint: {}", stake_pool.pool_mint);
+        }
```

Comment:
This approach doesn't seem right to me. I think this should all be implemented as Display and VerboseDisplay on a CliStakePool.
You can add additional information that should only appear in `command_list()` Display and VerboseDisplay responses to CliStakePool with the `#[serde(skip)]` tag, possibly with data wrapped in an Option to prevent extraneous calls in `command_list_all_pools()`. However, it's possible that will be complex enough that you actually need a different struct to do this simply (CliDetailedStakePool?).

Reply:
Yep Let's do it

Reply:
Done